specification/resourcehealth/resource-manager/readme.md

```diff
@@ -110,7 +119,17 @@ These settings apply only when `--tag=package-2020-05-01` is specified on the co
 input-file:
 - Microsoft.ResourceHealth/stable/2020-05-01/ResourceHealth.json
 ```
-
+### Suppression
+``` yaml
+directive:
+  - suppress: DefaultErrorResponseSchema
+    from: ResourceHealth.json
+    reason: Changing the Error Response would be considered a breaking change
+  - suppress: DefinitionsPropertiesNamesCamelCase
+    from: ResourceHealth.json
+    where: $.definitions.recommendedAction.properties["ActionUrl.Comment"]
```

Comment:
```suggestion
    where: $.definitions.recommendedAction.properties["_ActionUrl.Comment"]
```
Shouldn't that property have an underscore in front? Maybe this is why it isn't being honored.

Reply:
yeah i did try the above format. just tried the one without the underscore last.  i've added the underscore back

Reply:
hmm after making the change i don't see the error pop up as a lint diff. here is the original commit where I used the underscore https://github.com/Azure/azure-rest-api-specs/pull/22721/commits/809c958cf6febd52236c13459b724752f3d82768 

I may have messed up and thought it was still giving me the error. I apologies for raising this issue . Thank you verry much for your help!

Reply:
The original commit was missing `.properties`

Reply:
ah got it thank you again!